crates/noirc_evaluator/src/ssa/value.rs

```diff
@@ -0,0 +1,115 @@
+use crate::ssa::node::NodeId;
+use iter_extended::vecmap;
+use noirc_frontend::monomorphization::ast::Type;
+
+/// A `Value` is the descriptor used for items in the SSA IR.
```

Comment:
This is not accurate, `Value`s are a ssa_gen concept only as they are only used internally while creating the ssa IR to represent structs and tuples. Once the ssa_gen pass is complete there will be no more Values, only NodeIds and occasionally arrays of NodeIds.

Reply:
This has been fixed